extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -691,9 +695,20 @@ export class AmpStoryPage extends AMP.BaseElement {
       if (this.isBotUserAgent_) {
         mediaEl.muted = false;
         mediaEl.removeAttribute('muted');
+        if (mediaEl.tagName === 'AUDIO' && mediaEl.paused) {
+          mediaEl.play();
+        }
       } else {
-        return mediaPool.unmute(
-            /** @type {!./media-pool.DomElementDef} */ (mediaEl));
+        mediaEl = /** @type {!./media-pool.DomElementDef} */ (mediaEl);
+        const promises = [mediaPool.unmute(mediaEl)];
+
+        // Audio element on the first page of a story would not be playing since
```

Comment:
nit: it's not necessarily just the first page, it could potentially be any page where the browser has rejected playback for whatever reason.  Not sure if it's worth trying to message that here, for future readers